lib/game_setting/game_settings.rb

```diff
@@ -3,24 +3,26 @@
 require_relative './setting_types'
 
 module GameSettings
+    extend self
+
     include Messages
     include Validator
     include SettingTypes
 
     def opponent_type
         {
-            :name => OPPONENT_TYPE_SETTING,
-            :message => opponent_type_msg,
-            :valid? => -> (input: nil) { opponent_type_valid?(input: input) },
+            :name => SettingTypes::OPPONENT_TYPE_SETTING,
+            :message => Messages.opponent_type_msg,
+            :valid? => -> (input: nil) { Validator.opponent_type_valid?(input: input) },
             :parse => -> (input:) { input },
         }
     end
 
     def board_size
         {
-            :name => BOARD_SIZE,
-            :message => board_size_msg,
-            :valid? => -> (input: nil) { board_size_valid?(input: input) },
+            :name => SettingTypes::BOARD_SIZE,
+            :message => Messages.board_size_msg,
```

Comment:
I noticed a lot of these types of methods went from `something_msg` to `Messages.something_msg`.  In my head this both makes sense and doesn't (on one hand, `Messages something message`, on the other hand, `The something message in Messages`).  What were your thoughts, @serpient?

Reply:
Now that I can use self to show the overall module, i think it'll be cleaner to rename all of them to something like `Messages.board_size`

Reply:
Makes sense to me